Core/OxidEeEvents.php

```diff
@@ -293,6 +296,24 @@ public static function createOrderTransactionTable()
         self::$oDb->Execute($sQuery);
     }
 
+    /**
+     * Creates the module's payment method meta data table
+     *
+     * @since 1.2.0
+     */
+    public static function createPaymentMethodMetaDataTable()
```

Comment:
The current implementation only adds meta data for a single table, in this case `oxpayments`. However, by adding the `MetaDataModel` trait to a class and specifying a dedicated table where the meta data should be stored, it can be used for any model.

In short: in this case the naming is correct in my opinion :)